### ISSUE TITLE
install-assets.sh change breaks travis

### ISSUE THREAD

**Issue by liggitt** [openshift/origin#5561]
This code is breaking on travis:
```
  while [[ "$tries" -gt 0 ]]; do
    $cmd &>> ${log_file} || rc=$?
    [[ "$rc" == "0" ]] && return 0
    ((tries--))
  done
```

https://travis-ci.org/openshift/origin/builds/88568895#L124

```
hack/install-assets.sh: line 20: [[: npm.log: syntax error: invalid arithmetic operator (error token is ".log")
[ERROR] Command 'npm install -g npm@2.1.14' failed with rc 0, logs:
0.12s$ make check-test TESTFLAGS="-p=4" TESTS="''"
```

**Comment by liggitt** [comment]
Broken in https://github.com/openshift/origin/pull/5553
@sosiouxme PTAL

**Comment by sosiouxme** [comment]
Sorry. Fixing.

**Comment by jwforres** [comment]
Fixed by #5638
